sdk/core/azure-core-test/src/main/java/com/azure/core/test/MyRestException.java

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.test;
+
+import com.azure.core.exception.HttpResponseException;
+import com.azure.core.http.HttpResponse;
+
+public class MyRestException extends HttpResponseException {
+    /**
+     * Creates a service request exception.
+     *
+     * @param message Message associated with exception.
+     * @param response HttpResponse associated with the exception.
+     * @param body Deserialized body of {@code response}.
+     */
+    public MyRestException(String message, HttpResponse response, HttpBinJSON body) {
+        super(message, response, body);
+    }
+
+    @Override
+    public HttpBinJSON getValue() {
+        return (HttpBinJSON) super.getValue();
+    }
+}
```

Comment:
I'm not sure why this class exists?

Reply:
I moved it from azure-core to azure-core-test. It's used in a bunch of tests.

Reply:
Ok. I can't recall off the top of my head if we publish the javadoc for this module, but if we do, I think we should stop it. This isn't an end-user facing library, so it shouldn't be included. If we do, please can you exclude that package?

Reply:
We used to publish javadocs for test module. I have removed it now.